src/components/order-service/order-service-info.html

```diff
@@ -27,7 +27,7 @@
     </div>
   </div>
   <div class="order-service-description-block">
-    <p ng-if="$ctrl.currentStep.id !== 'plans' && ($ctrl.selectedPlan.spec.externalMetadata.displayName || $ctrl.selectedPlan.spec.description)">
+    <p ng-if="$ctrl.currentStep.id !== 'plans' && !$ctrl.multipleServicePlans && ($ctrl.selectedPlan.spec.externalMetadata.displayName || $ctrl.selectedPlan.spec.description)">
```

Comment:
Just noticed this, but can't be remove the `$ctrl.currentStep.id !== 'plans'` check? This is always the info step.

Reply:
Yep.  Good catch!  Fixed.